env.example

```diff
@@ -35,6 +35,14 @@ LOG_LEVEL=debug
 # LOG_FILE is used to set a destination path to write logs. Works in production mode only.
 LOG_FILE=
 
+# Used to set if Pino should log are indexed to ElasticSearch, PINO_ELASTIC=1 to enable
+PINO_ELASTIC=
```

Comment:
`LOG_ELASTIC` maybe.  People won't understand what Pino means, but they do understand logs.

Reply:
What if someone sets `LOG_FILE` and `LOG_ELASTIC`?  Should we document which one gets priority?

Reply:
the code has LOG_ELASTIC as priority
`const logger = pino({ options }, process.env.LOG_ELASTIC ? streamToElastic : destination);` but I added a line to the `env.example` to be more explicit
```
# Used to set if Pino should log are indexed to ElasticSearch, LOG_ELASTIC=1 to enable, if LOG_FILE and LOG_ELASTIC is set,
# LOG_ELASTIC will take priority
```